Themis/Views/Assessment/CorrectionSidebar/FeedbackCellView.swift

```diff
@@ -18,7 +18,9 @@ struct FeedbackCellView: View {
     @State var feedback: AssessmentFeedback
     private var editingDisabled: Bool { assessmentVM.readOnly || !assessmentVM.allowsInlineFeedbackOperations }
     private var tapGestureDisabled: Bool { feedback.scope != .inline || !assessmentVM.allowsInlineFeedbackOperations }
-    
+    private var feedbackText: String {feedback.baseFeedback.text ?? feedback.baseFeedback.testCase?.testName ?? "Feedback"}
```

Comment:
As an additional improvement, you could check if the feedback is automatic or manual and call that "Test case" for automatic feedback.

Reply:
Thanks for the suggestion 👍